README.md

```diff
@@ -42,51 +42,59 @@ Your code interview will be evaluated based on your repo, so make sure all files
 
-### Task 2: Data Aggregation
+### Python Task 2: Data Aggregation
 
-Create a time series table based on the `data/raw.csv` file we created from **Task 1** that has the following fields
+Write a process to produce a time series table based on the data you've pulled so far. It should have the following fields
 
-- `created_date_hour`: the timestap of request creation by date and hour
-- `complaint_type`: the type of the complaint
+- `created_time`: the timestap of request creation by date and hour OR just date
+- `complaint_type`: the type of the complaint (optional)
 - `count`: the count of service requests by `complaint_type` by `created_date_hour`
 
-Store this table in a csv under the `data` folder with a csv file name of your choice.
+The granularity of the `created_time` column and whether the data is broken down by complaint type should be controlled by parameters passed in from the command line. 
 
-### Task 3: Data Visualization
+Create as many versions of this table that you think is appropriate to show that your code works, and store these versions as appropriately named .csv files.
 
-Create a multi-line plot to show the total service request counts by `created_date_hour` for each `complaint_type`. Make sure you store the image of the plot in the `data` folder as a `.png` file.  
+### Python Task 3: Data Visualization
 
-### Task 4: Spatial data processing
+Write a bash/python process to create a line plot to show the total service request counts by `created_time` that breaks down the data by `complaint_type` if that column is present. Save as many `.png` files of your plots as you think is necessary to show us that your code works. 
+If you would like to go back and download different tables to visualize, that's totally fine. Chose whichever pulls you think make it easiest to show us your work. 
 
-At Data Engineering, we enhance datasets with geospatial attributes, such as point locations and administrative boundaries. To help us better understand the data from **Task 1**, we would like you to join the initial raw data to the **[2020 NTA (Neighborhood Tabulation Area) boundaries](https://www1.nyc.gov/site/planning/data-maps/open-data/census-download-metadata.page)** and create a choropleth map of 7 day total count by NTA of a specific `complaint_type` of your choice.
+### Python Task 4: Spatial data processing
 
-Depending on how you generate the map, you can store the map as a `.png` or `.html` under the `data` folder.
+At Data Engineering, we enhance datasets with geospatial attributes, such as point locations and administrative boundaries. To help us better understand the data from **Python Task 1**, we would like you to join the initial raw data to the some NYC administraive boundary of your chosing. Then create a choropleth map of count of complaints of a specific `complaint_type` of your choice.
```

Comment:
change `some NYC administraive boundary` to `an NYC administraive boundary`
Given how prevalent `complaint_type` is throughout the challenge, I don't think that we can make it optional (i.e. complaints of a specific `complaint_type` of your choice.)

Reply:
Oh I meant optional w/r/t the code, not optional within the scope of the data challenge. I think it's a common pattern to leave an argument out and have the code default to "all". We can be clearer that the interviewee needs to save at least one file that does break down the data by complaint type